From 40ec2100ad2bc73e6912e936b33fdd5f17f9ef96 Mon Sep 17 00:00:00 2001
Message-Id: <40ec2100ad2bc73e6912e936b33fdd5f17f9ef96.1411497349.git.jen@redhat.com>
In-Reply-To: <c1a65d1a3f9888636f1026da1d5f77d985a8815b.1411497349.git.jen@redhat.com>
References: <c1a65d1a3f9888636f1026da1d5f77d985a8815b.1411497349.git.jen@redhat.com>
From: Juan Quintela <quintela@redhat.com>
Date: Tue, 16 Sep 2014 15:50:18 -0400
Subject: [CHANGE 2/7] migration: set speed to maximum during last stage
To: rhvirt-patches@redhat.com,
    jen@redhat.com

RH-Author: Juan Quintela <quintela@redhat.com>
Message-id: <1410882623-10906-3-git-send-email-quintela@redhat.com>
Patchwork-id: 61185
O-Subject: [PATCH qemu-kvm RHEL6.6 2/7] migration: set speed to maximum during last stage
Bugzilla: 1142756 970103
RH-Acked-by: Dr. David Alan Gilbert (git) <dgilbert@redhat.com>
RH-Acked-by: Amit Shah <amit.shah@redhat.com>
RH-Acked-by: Markus Armbruster <armbru@redhat.com>

Signed-off-by: Juan Quintela <quintela@redhat.com>
Signed-off-by: Jeff E. Nelson <jen@redhat.com>
---
 migration.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/migration.c b/migration.c
index 0400edb..e29262d 100644
--- a/migration.c
+++ b/migration.c
@@ -464,6 +464,7 @@ void migrate_fd_put_ready(void *opaque)
 
         bdrv_drain_all();
         bdrv_flush_all();
+        qemu_file_set_rate_limit(s->file, SIZE_MAX);
         if ((qemu_savevm_state_complete(s->mon, s->file)) < 0) {
             if (old_vm_running) {
                 vm_start();
-- 
1.9.3

